app/wallet/control.js

```diff
@@ -48,14 +48,17 @@ export const importPrivateKey = (walletService, passphrase, accountNum, wif, res
     walletService.importPrivateKey(request, (err, res) => err ? fail(err) : ok(res));
   });
 
-export const importScript = (walletService, passphrase, script, rescan, scanFrom) =>
+export const importScript = (walletService, passphrase, script, isWatchingOnly) =>
   new Promise((ok, fail) => {
     const request = new api.ImportScriptRequest();
     request.setPassphrase(new Uint8Array(Buffer.from(passphrase)));
     request.setScript(new Uint8Array(Buffer.from(hexToBytes(script))));
-    request.setRescan(rescan);
-    request.setScanFrom(scanFrom);
-    request.setRequireRedeemable(true);
+    request.setRescan(false);
+    request.setScanFrom(0);
+    // watch only wallets are not redeemable of their scripts, therefore we
+    // don't require them to be redeemable.
+    const isRequireRedeem = isWatchingOnly ? false : true;
+    request.setRequireRedeemable(isRequireRedeem);
```

Comment:
This isn't true. The "redeemable" part checks whether one of the addresses of the multisig belogs to the wallet, irrespective of whether the wallet has the private keys for that address or not.

So I think this should remain `true` even for watching only wallets since it ensures the corresponding (cold) wallet will have the keys for at least one of the pubkeys in the script.

Reply:
Yeah, you are right. Fixed